src/Gitlab/Client.ts

```diff
@@ -134,6 +134,17 @@ export class GitLabClient {
         }
     }
 
+    public async getProjectAccessLevel(id: ProjectId): Promise<AccessLevel> {
+        try {
+            const me = await this.user();
```

Comment:
I wish there was some type and value validation in `user()`.
This flow just assumes that me is a valid object and has a valid attribute `id`.

Given it's all wrapped in a `catch (err) {console.warn(err)}`, it might be ok for debugging, in case the API breaks...

Reply:
I think it's okay for now, though point made.